lighthouse-core/gather/driver.js

```diff
@@ -717,10 +717,20 @@ class Driver {
    * @return {string}
    */
   getRequestContent(requestId) {
-    return this.sendCommand('Network.getResponseBody', {
-      requestId,
-    // Ignoring result.base64Encoded, which indicates if body is already encoded
-    }).then(result => result.body);
+    return new Promise((resolve, reject) => {
+      // If this takes more than 2s, reject the Promise.
```

Comment:
maybe add a note with link back to your investigation for why this method in particular can be problematic?